### PR TITLE
Add tests for golang dwarf

### PR DESCRIPTION
**Pull request by yota9**
For the 267 and 268 PRs

### CODE REVIEW

File: test/AArch64/go_dwarf.test [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Check that the llvm-bolt update the dwarf information correctly in case:
+#  - DW_AT_low_pc is nullified in case the DW_AT_ranges tag already exists
+#  - DW_AT_high_pc is in the form of DW_FORM_addr
+
+RUN: llvm-bolt %p/Inputs/gcgort_aarch64 -o %t -update-debug-sections
```
Comment (aaupov) [inline test/AArch64/go_dwarf.test:5]: Can you please also add checks for the values in the original binary? So that we know they are updated correctly.

**Review by aaupov — COMMENTED** [review]
LGTM
@ayermolo: can you please take a look if the test looks OK? (doesn't miss anything important)

**Comment by aaupov** [comment]
@yota9:
Please add references to PRs to the commit message (because PR message is lost after merge)

File: test/AArch64/go_dwarf.test (hunk 1 above)
Comment (yota9) [reply test/AArch64/go_dwarf.test:5]: Sure, done

**Comment by ayermolo** [comment]
lgtm

**Comment by yota9** [comment]
@ayermolo I was trying to play-around this problem but didn't succeed.. It is like "hello-world" golang test average size. AFAIU most of the size is going from the golang runtime code, but the default gc seems to be unable to link it in shared library. Anyway I will re-use this test (and add a couple more) with golang support.

**Comment by yota9** [comment]
The DWARF test was made based on the small C hello-world, no need for this binaries for now

**Pull request closed without merging**